Sources/Hub/HubApi.swift

```diff
@@ -8,25 +8,29 @@
 import Foundation
 
 public struct HubApi {
-    var downloadBase: URL
-    var hfToken: String?
-    var endpoint: String
+    public var downloadBase: URL
+    public var hfToken: String?
+    public var endpoint: String
```

Comment:
```suggestion
    public let downloadBase: URL
    public let hfToken: String?
    public let endpoint: String
```

I think it's ok to make them public, but then I'd make them immutable